server.go

```diff
@@ -245,10 +245,19 @@ func (s *Server) register(sd *ServiceDesc, ss interface{}) {
 	s.m[sd.ServiceName] = srv
 }
 
-// ServiceInfo contains method names and metadata for a service.
+// MethodInfo contains information about an RPC.
```

Comment:
MethodInfo contains the information of an RPC including its method name and type.